lib/vsc/utils/generaloption.py

```diff
@@ -29,25 +29,31 @@
 @author: Stijn De Weirdt (Ghent University)
 @author: Jens Timmerman (Ghent University)
 """
-
-import ConfigParser
 import copy
 import difflib
 import inspect
 import operator
 import os
 import re
-import StringIO
 import sys
 import textwrap
+from functools import reduce
 from optparse import OptionParser, OptionGroup, Option, Values
 from optparse import BadOptionError, SUPPRESS_USAGE, OptionValueError
 from optparse import SUPPRESS_HELP as nohelp  # supported in optparse of python v2.4
 from optparse import gettext as _gettext  # this is gettext.gettext normally
+
+try:
```

Comment:
no, please use a `is_py3` function or at the very least the sys.version crap

Reply:
done